public/img/cvlogo.svg

```diff
@@ -0,0 +1 @@
43-2.89,1.59-3,3.9s-.11,4.37-.09,6.55c0,.84-.14,1.24-1.11,1.13a26.56,26.56,0,0,0-3.46-.11c-.72,0-.85-.3-.84-.93,0-4.57,0-9.15,0-13.72,0-.71.16-1,.93-1a20.05,20.05,0,0,0,3.1,0c1-.08,1.4.24,1.23,1.24A8.92,8.92,0,0,0,118,33.62Z"/><path class="cls-1" d="M109.64,39.29c0,2.27,0,4.54,0,6.81,0,.7-.15,1-.92,1-1.16-.07-2.32,0-3.48,0-.57,0-.87-.11-.87-.78,0-4.7,0-9.4,0-14.1,0-.56.19-.8.76-.79,1.23,0,2.47,0,3.71,0,.66,0,.79.28.78.86C109.63,34.59,109.64,36.94,109.64,39.29Z"/><path class="cls-1" d="M161.51,39.07c0-2.23.05-4.45,0-6.68,0-.92.27-1.16,1.15-1.1,1.11.07,2.23,0,3.35,0,.57,0,.75.18.75.75q0,7.1,0,14.18c0,.62-.19.85-.82.83-1.16,0-2.31-.05-3.47,0-.74,0-1-.25-.95-1C161.53,43.76,161.51,41.41,161.51,39.07Z"/><path class="cls-3" d="M166.73,27.51A2.38,2.38,0,0,1,164,30.07a2.53,2.53,0,0,1-2.78-2.58,2.56,2.56,0,0,1,2.8-2.57A2.4,2.4,0,0,1,166.73,27.51Z"/><path class="cls-4" d="M106.88,30.07a2.55,2.55,0,0,1-2.71-2.69A2.62,2.62,0,0,1,107,24.92a2.58,2.58,0,0,1,2.76,2.65A2.54,2.54,0,0,1,106.88,30.07Z"/><path class="cls-5" d="M38.18,28.35c0-2.07,0-4.14,0-6.21,0-.75.16-1.05,1-1,3.35,0,6.7,0,10.05,0a1,1,0,0,1,1.17.87,66.79,66.79,0,0,1,2,12.34c.07,1-.17,1.31-1.21,1.29-3.91-.06-7.83-.06-11.73,0-1,0-1.27-.32-1.24-1.28C38.23,32.33,38.18,30.34,38.18,28.35Z"/><path class="cls-5" d="M38.06,46.05c0-1.95,0-3.9,0-5.85,0-.88.16-1.25,1.17-1.24,4,.07,8,0,12,0,.82,0,1.11.2,1.07,1.07a66.3,66.3,0,0,1-1.82,12.13c-.21.91-.58,1.17-1.51,1.16-3.27-.07-6.54-.06-9.81,0-.83,0-1.1-.22-1.07-1.06C38.1,50.19,38.06,48.12,38.06,46.05Z"/><path class="cls-5" d="M62.54,39c2,0,3.92,0,5.88,0,.82,0,1,.27.94,1A35.94,35.94,0,0,1,65.75,52.8a1.11,1.11,0,0,1-1.15.66c-3.39,0-6.79,0-10.18,0-.74,0-.85-.18-.67-.88a56.47,56.47,0,0,0,1.81-12.26c0-1,.36-1.43,1.47-1.36C58.86,39.07,60.71,39,62.54,39Z"/><path class="cls-5" d="M62.51,35.4c-1.91,0-3.83-.07-5.74,0-1,.06-1.18-.44-1.22-1.24a64,64,0,0,0-1.5-10.85c-.53-2.28-.51-2.28,1.86-2.28,2.8,0,5.59,0,8.38,0a1.43,1.43,0,0,1,1.5.86,35.46,35.46,0,0,1,3.56,12.54c.1.84-.22,1-1,1C66.42,35.38,64.47,35.4,62.51,35.4Z"/><path class="cls-5" d="M38.06,63.59c0-2,0-3.91,0-5.86,0-.67.2-.9.89-.89,3.07,0,6.15,0,9.22,0,.67,0,.73.21.52.77A46,46,0,0,1,41.86,70a5.52,5.52,0,0,1-3.49.63c-.41-.23-.3-.62-.3-1Q38.06,66.64,38.06,63.59Z"/><path class="cls-5" d="M48.88,17.64c-3.3,0-6.61,0-9.91,0-.59,0-.73-.25-.73-.79q0-6.15-.07-12.28c0-.63.18-.83.82-.79a4.42,4.42,0,0,1,4,2A58.54,58.54,0,0,1,48.6,16.44,2,2,0,0,1,48.88,17.64Z"/><path class="cls-5" d="M46.72,69.07A57.9,57.9,0,0,0,52.2,57.5c.15-.41.34-.66.84-.66,3.31,0,6.62,0,10.17,0A32.37,32.37,0,0,1,46.72,69.07Z"/><path class="cls-5" d="M63.25,17.71H53.14a.8.8,0,0,1-.9-.57,64.51,64.51,0,0,0-5.55-11.6A32.92,32.92,0,0,1,63.25,17.71Z"/><path class="cls-6" d="M209.06,37.19c-.48,0-.95,0-1.43,0-1.25,0-1.37-.2-.82-1.35a2.32,2.32,0,0,1,2.43-1.33,2.24,2.24,0,0,1,2.19,1.68c.23.67.14,1.08-.71,1C210.17,37.16,209.61,37.19,209.06,37.19Z"/><path class="cls-7" d="M254.57,37.28h-1.31c-1.24,0-1.35-.2-.79-1.36a2.48,2.48,0,0,1,4.74.8c.06.46-.1.57-.49.56Z"/></g></g></svg>
```

Comment:
Are we not having the logo in the assets folder?

Reply:
The logo that was there in the asset folder was of png format & had a white bg which is not the white bg of the readme.MD , so I had used an svg with a transparent bg, still there are some other whites in the svg, I can use the circuitversenew.png if you want @satu0king , it has a complete transparent bg.